frontend/src/main/web/lib/constants/Configs.js

```diff
@@ -0,0 +1,8 @@
+var Configs = {
+  baseUrl: null,
+  user: null, // see org.zanata.rest.editor.dto.User
+  data: null,
+  urlPostfix: ''
+};
```

Comment:
Since all the config is empty initially, would be good to see a short explanation of why. I think this is stuff that gets read off a DOM node before React takes over its rendering, is that right?